src/System.Security.Cryptography.Xml/tests/RSAKeyValueTest.cs

```diff
@@ -96,7 +96,16 @@ public void LoadXml_InvalidXml(string xml)
             xmlDocument.LoadXml(xml);
 
             RSAKeyValue rsa = new RSAKeyValue();
-            Assert.Throws<CryptographicException>(() => rsa.LoadXml(xmlDocument.DocumentElement));
+
+            // FormatException exception because desktop does not
+            // check if Convert.FromBase64String throws
+            // Related to: https://github.com/dotnet/corefx/issues/18690
```

Comment:
I don't know @bartonjs 's practice but I prefer not to have issue numbers in the code as it becomes a mess of historical notes. Put the issue in the commit message, it's easy to find later with 'blame'.